### DISCUSSION TITLE
Inquiry Regarding Wiki.js 3.0 Usability and Error Resolution

### DISCUSSION THREAD

**Discussion by yanzhen1031** [requarks/wiki#7303] [Help / Questions] [answered]
Hi,
﻿
I hope you're doing well. I’m currently working with Wiki.js 3.0.0-alpha.363, and I’ve encountered some issues during setup, specifically related to missing dependencies like the pako package. Before I proceed further, I wanted to ask: is version 3.0 ready for use, or is it still under active development? Do you recommend using this version at this stage, and are the errors I'm encountering something that I can resolve on my own, or should I wait for a more stable release?

![image](https://github.com/user-attachments/assets/d9597656-093a-4055-b4b6-5da2cd42a261)

 
Thank you for your guidance!

**Comment by francoisauclair911** [comment, marked as answer]
Wiki.js 3.0 is still in the alpha stage, which means it’s under active development and not yet stable. Alpha versions are typically released for testing and feedback, and they often contain bugs, missing features, and dependencies that may not be fully resolved. The issues you’re encountering, like the missing pako package, are likely due to the unfinished nature of this release.

Given that you’re running into setup issues, it’s probably best to wait for a more stable beta or final release unless you’re comfortable troubleshooting these types of errors and contributing to the development by reporting bugs or fixing issues. If you need a more reliable setup, I recommend sticking with the latest stable version of Wiki.js.

**Monitor the releases via https://github.com/requarks/wiki/releases/**
